### ISSUE TITLE
The training videos

### ISSUE THREAD

**Issue by linchuming** [sniklaus/pytorch-sepconv#17]
Hi, thanks for your sharing. I like your work. 
Recently I want to retrain your model, and I download some YouTube videos. But I train the model which can not achieve the PSNR 35.73 dB in the Middlebury. So can you offer your training videos list?
Thanks for you very much!

**Comment by PanXie** [comment]
@linchuming ，can u share how to define the backword function of SeparableConvolution layer ? thx

**Comment by LINSEVENQI** [comment]
Excuse me, can you share with me your training set?

**Comment by gooodpower** [comment]
@linchuming Hi, How you download some YouTube videos? I see the homepage of youtube8m( https://research.google.com/youtube8m/download.html), but what I download is tfrecords. How can I download the original videos(not tfrecords)？
Thanks very much！

**Comment by sniklaus** [comment]
I have been using the Vimeo90k dataset in our most recent paper on softmax splatting, which achieves state-of-the-art performance: http://sniklaus.com/softsplat

As such, I would recommend everyone who is interested in training data for video frame interpolation to give Vimeo90k a try: http://toflow.csail.mit.edu/
